docs/modules/operation/pages/deep-dive/admin/scv-configuration.adoc

```diff
@@ -1,52 +1,67 @@
+
 [[scv-overview]]
 = Secure Credentials Vault
 
-Secure Credentials Vault (SCV) lets you encrypt credentials the {page-component-title} system uses.
-SCV uses a Java KeyStore (JKS) to store credentials in JKS format.
-SCV is enabled by default on {page-component-title}.
+The secure credentials vault (SCV) lets you encrypt credentials used by {page-component-title}.
+It uses Java KeyStore (JKS) to store credentials in the JKS format.
```

Comment:
I don't fully understand this technology, but wonder if writing it like this is still accurate. The current sentence seems redundant, a bit like saying "It uses the Portable Document Format to store files in PDF format." Chandra would be able to confirm, though.

```suggestion
It stores credentials in the Java KeyStore (JKS) format.
```

Reply:
I'm not sure either. I can send this PR to him for review.

Reply:
Your suggestion is good.